sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -172,41 +176,53 @@ You'll need to create an asynchronous [`ServiceBusReceiverAsyncClient`][ServiceB
 either a queue or a topic subscription.
 
+By default, the receive mode is [`ReceiveMode.PEEK_LOCK`][ReceiveMode]. This tells the broker that the receiving client
+wants to settle received messages explicitly. The message is made available for the receiver to process, while held
+under an exclusive lock in the service so that other, competing receivers cannot see it.
+`ServiceBusReceivedMessage.getLockedUntil()` indicates when the lock expires and can be extended by the client using
+`receiver.renewMessageLock()`.
+
 #### Receive a batch of messages
 
-The snippet below creates a `ServiceBusReceiverClient` to receive messages from a topic subscription. It returns a batch
-of messages when 10 messages are received or until 30 seconds have elapsed, whichever happens first.
+The snippet below creates a [`ServiceBusReceiverClient`][ServiceBusReceiverClient] to receive messages from a topic
+subscription.
 
-<!-- embedme ./src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java#L81-L94 -->
+<!-- embedme ./src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java#L84-L101 -->
 ```java
 ServiceBusReceiverClient receiver = new ServiceBusClientBuilder()
     .connectionString("<< CONNECTION STRING FOR THE SERVICE BUS NAMESPACE >>")
     .receiver()
     .topicName("<< TOPIC NAME >>")
     .subscriptionName("<< SUBSCRIPTION NAME >>")
-    .receiveMode(ReceiveMode.PEEK_LOCK)
     .buildClient();
 
+// Receives a batch of messages when 10 messages are received or until 30 seconds have elapsed, whichever
+// happens first.
 IterableStream<ServiceBusReceivedMessageContext> messages = receiver.receive(10, Duration.ofSeconds(30));
 messages.forEach(context -> {
     ServiceBusReceivedMessage message = context.getMessage();
     System.out.printf("Id: %s. Contents: %s%n", message.getMessageId(),
         new String(message.getBody(), StandardCharsets.UTF_8));
 });
+
+// When you are done using the receiver, dispose of it.
+receiver.close();
 ```
 
 #### Receive a stream of messages
 
-The asynchronous `ServiceBusReceiverAsyncClient` continuously fetches messages until the `subscription` is disposed.
+The asynchronous [`ServiceBusReceiverAsyncClient`][ServiceBusReceiverAsyncClient] continuously fetches messages until
+the `subscription` is disposed.
 
-<!-- embedme ./src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java#L101-L115 -->
+<!-- embedme ./src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java#L108-L129 -->
 ```java
 ServiceBusReceiverAsyncClient receiver = new ServiceBusClientBuilder()
     .connectionString("<< CONNECTION STRING FOR THE SERVICE BUS NAMESPACE >>")
     .receiver()
     .queueName("<< QUEUE NAME >>")
     .buildAsyncClient();
 
+// receive() operation continuously fetches messages until the subscription is disposed.
 Disposable subscription = receiver.receive().subscribe(context -> {
```

Comment:
continuously  ->  `infinitely`
This will register in user mind that they have to handle this properly.